workorders.lic

```diff
@@ -286,28 +286,63 @@ class WorkOrders
       stock_needed = 1
       order_rem(info['stock-room'], stock_needed, info['stock-number'])
     when 'You get'
-      /are (.*) parts left of the water/ =~ bput('count my water', 'are .* parts left of the water')
+      /are (.*) parts left of the water|only (.*) part left/ =~ bput('count my water', 'are .* parts left of the water')
```

Comment:
Hello,

I could not get the way mentioned to work so I have it do it twice to 
capture it. If a better way exists, please add. This did not work:

/are (.*) parts left of the water|only (.*) part left/ =~ bput('count my 
water', 'are .* parts left of the water', 'only .* part left')
       existing = Regexp.last_match(1).scan(/\w+/).to_a

-- 
Regards,
Dr. Walter Wesley "Wes" Snyder V
484-300-4833 (h)
484-431-3355 (m)
484-302-7006 (f)

On 09/04/2016 10:50 PM, Robert Herbig wrote:
>
> In workorders.lic 
> <https://github.com/rpherbig/dr-scripts/pull/737#discussion_r77465953>:
>
> > @@ -286,28 +286,63 @@ class WorkOrders
> >        stock_needed = 1
> >        order_rem(info['stock-room'], stock_needed, info['stock-number'])
> >      when 'You get'
> > -      /are (.*) parts left of the water/ =~ bput('count my water', 'are .* parts left of the water')
> > +      /are (.*) parts left of the water|only (.*) part left/ =~ bput('count my water', 'are .* parts left of the water')
>
> Why does this output |count my water| twice (here and line 291)? 
> Should one of these be |count my second water|?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub 
> <https://github.com/rpherbig/dr-scripts/pull/737/files/7aaafb3fc3d1daa463a35a97157ca897a5e00fd4#r77465953>, 
> or mute the thread 
> <https://github.com/notifications/unsubscribe-auth/AU8XXIUSuanhA80owa4Z6ZV0ZcjMB_Lyks5qm4NqgaJpZM4J0roH>.
>